config/lura.go

```diff
@@ -0,0 +1,70 @@
+package config
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+
+	"github.com/luraproject/lura/v2/config"
+)
+
+const (
+	// Namespace is the key under the Lura's "extra_config" root
+	// section, for a valid config. See [config] documentation for
+	// details.
+	Namespace = "telemetry/opentelemetry"
+)
+
+var (
+	// ErrNoConfig is used to signal no config was found
+	ErrNoConfig = errors.New("No config found for opentelemetry")
+)
+
+// FromLura extracts the configuration from the Lura's ServiceConfig
+// "extra_config" field.
+//
+// In case no "Layers" config is provided, a set of defaults with
+// everything enabled will be used.
+func FromLura(srvCfg config.ServiceConfig) (*Config, error) {
+	cfg := new(Config)
+	tmp, ok := srvCfg.ExtraConfig[Namespace]
+	if !ok {
+		return nil, ErrNoConfig
+	}
+	buf := new(bytes.Buffer)
+	json.NewEncoder(buf).Encode(tmp)
+	if err := json.NewDecoder(buf).Decode(cfg); err != nil {
+		return nil, err
+	}
+
+	if cfg.Layers == nil {
```

Comment:
If we customize a layer, the rest will be disabled, that can be confusing since not setting anything enables everything by default. May be, each layer should be checked independently. We can also reverse the option so the default struct value (`false`) mimics this behaviour.